### PR TITLE
add: instructions and guide related to listmonk.fossunited.org

### PR DESCRIPTION
**Pull request by Mangeshrex**
This PR is to add documentation and all the instructions related to how Listmonk is self hosted for FOSS United on the VPS. 

It provides information related to setup directories, ports, configuration files, backups and etc.

### CODE REVIEW

**Review by rahulporuri — APPROVED** [review]
I think this looks good. Will be good to get one more pair of eyeballs on this, if possible.

**Comment by Mangeshrex** [comment]
Cool, requested reviews from Riya.

File: servers/apps/listmonk/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+[Listmonk](https://listmonk.app) is a bulk email sending service which is used heavily at FOSS United for sending out bulk emails to attendees, speakers and in general to the whole FOSS Community.
+
+Server: Listmonk (Nanode: 1GB) 
+App: Listmonk
+Latest updated version: ```v3.0.0 (f9120d9 2024-02-04T11:20:27Z, linux/amd64)```
+Install Dir: `/opt/listmonk`
+Backups Dir: `/opt/backups`
+OS: Ubuntu 20.04 LTS
+RAM: 1Gib
```
Comment (RiyaMathew-11) [inline servers/apps/listmonk/README.md:11]: RAM is 1GB not Gib(Gibibyte)?

**Review by RiyaMathew-11 — CHANGES_REQUESTED** [review]

File: servers/apps/listmonk/README.md
Comment (RiyaMathew-11) [inline servers/apps/listmonk/README.md:1]: @Mangeshrex it's mentioned that now we have auto updates - would be nice to document how it was done before. Any processes or commands or even useful links to refer to.

File: servers/apps/listmonk/config.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+# IMPORTANT: Leave both values empty to disable authentication on admin
+# only where an external authentication is already setup.
+admin_username = "listmonk"
+admin_password = [admin_password]
+
+# Database.
+[db]
+host = "localhost"
+port = 5432
+user = "listmonk"
```
Comment (RiyaMathew-11) [inline servers/apps/listmonk/config.toml:19]: Nitpicky - but [password] seems fine compared to [gooby-wooby-password]

**Review by RiyaMathew-11 — COMMENTED** [review]
Added a few tangential comments too

File: servers/apps/listmonk/README.md (hunk 1 above)
Comment (Mangeshrex) [reply servers/apps/listmonk/README.md:11]: Ohh cool, will update it. It seems, I was wrong all this time about GiB being GigaByte. As `cfdisk` command in linux calculates Gigabytes as GiB. 

Dumb thing, but yeah fine.

**Review by RiyaMathew-11 — APPROVED** [review]

File: servers/apps/listmonk/README.md
Comment (RiyaMathew-11) [reply servers/apps/listmonk/README.md:1]: If I am understanding this correctly - the cron-job covers the auto-update to happen right ? What I mentioned here is context on how you used to do updates before - manually I presume ?

**Pull request merged**